_includes/tutorials/rekeying-function/ksql/code/tutorial-steps/dev/continuous-regex.sql

```diff
@@ -0,0 +1,8 @@
+CREATE STREAM customers_with_area_code AS
+    SELECT 
+      customers.rowkey as id,
+      firstname,
+      lastname,
+      phonenumber,
+      REGEXREPLACE(phonenumber, '\\(?(\\d{3}).*', '$1') as area_code
+    FROM customers;
```

Comment:
this intermediate stream is necessary in 0.7.0+, when expression support was added to `PARTITION BY` 

cf. https://github.com/confluentinc/ksql/pull/4032